### PR TITLE
Adding in documentation for newly added user states fields, in additi…

### PR DESCRIPTION
**Pull request by godelstheory**
Adding in documentation for newly added activity segments. In addition, updating documentation for `clients_last_seen` to provide field descriptions for several high-value fields.

### CODE REVIEW

File: src/datasets/bigquery/clients_last_seen/reference.md [hunk 1]
```diff
@@ -60,6 +60,41 @@ The data is available in Re:dash and BigQuery. Take a look at this full running
 
 # Data Reference
 
+## Field Descriptions 
+The [“*_bits” fields](../../../cookbooks/clients_last_seen_bits.md) store the relevant activity of a client in a 28 day 
+window, as a 28 bit integer. 
+For each bit, a 1 corresponds to the specific activity occurring on that day.
+
+### Activity Segment/User State/Core Active Specific
```
Comment (irrationalagent) [inline src/datasets/bigquery/clients_last_seen/reference.md:68]: do we want to add a link to relevant chapter on user segments here?

File: src/datasets/bigquery/clients_last_seen/reference.md [hunk 2]
```diff
@@ -60,6 +60,41 @@ The data is available in Re:dash and BigQuery. Take a look at this full running
...
+
+* `is_core_active_v1`: Boolean indicating if the client satisfies conditions of being core active on that day. 
+* `activity_segments_v1`: The activity segment applicable to the client that day.
+* `is_regular_user_v3`:  Boolean indicating if the client satisfies conditions of being regular user on that day.
```
Comment (irrationalagent) [inline src/datasets/bigquery/clients_last_seen/reference.md:72]: nit: "...of being A regular user"

Comment (irrationalagent) [reply src/datasets/bigquery/clients_last_seen/reference.md:72]: same thing next two lines

File: src/datasets/bigquery/clients_last_seen/reference.md [hunk 3]
```diff
@@ -60,6 +60,41 @@ The data is available in Re:dash and BigQuery. Take a look at this full running
...
+
+### New Profile Specific
+* `first_seen_date`: Date the client sent their first main ping. 
+* `second_seen_date`: Date the client sent their first main ping.
+* `days_since_first_seen`: Number of days since `first_seen_date`
+* `days_since_second_seen`: Number of days since `second_seen_date`
+* `new_profile_5_day_activated_v1`: Boolean indicating if a new profile has sent a ping 5 out of their first 7 days.
+* `new_profile_14_day_activated_v1`: Boolean indicating if a new profile has sent a ping 8 out of their first 14 days.
+* `new_profile_21_day_activated_v1`: Boolean indicating if a new profile has sent a ping 12 out of their first 21 days.
+* `days_since_created_profile`: Number of days since the profile creation date.
```
Comment (irrationalagent) [inline src/datasets/bigquery/clients_last_seen/reference.md:96]: Might be worth calling out that this field is non-null only when the value is 27 or less. `profile_age_in_days` should give the profile age for all clients that have a PCD

**Review by irrationalagent — APPROVED** [review]
Just a few nits. Thanks for doing this Corey

File: src/concepts/segments.md [hunk 4]
```diff
@@ -135,6 +163,40 @@ WHERE
     cd.submission_date BETWEEN '2020-01-01' AND '2020-03-01'
 ```
 
+Using _is_core_active_v1_:
+
+Here are two basic ways to calculate a times series that counts the number of clients qualifying as Core Active:
```
Comment (felixlawrence) [inline src/concepts/segments.md:168]:
```suggestion
Here are two basic ways to calculate a time series that counts the number of clients qualifying as Core Active:
```

**Review by felixlawrence — APPROVED** [review]
This documentation LGTM. I'm not reviewing the definitions themselves because I believe you're already aware of my suggested potential improvements, the definitions are already settled, and as is they're good enough to be useful

I like how you dodge the ["how should we define MAU" discussion](https://docs.google.com/document/d/1OpejuD-wtViUX4nwddmaIHF85hU8WknjJBT-zjxxgkE) by describing "two basic ways to calculate a time series" and not giving that time series a name! Well played :stuck_out_tongue:

File: src/datasets/bigquery/clients_last_seen/reference.md (hunk 1 above)
Comment (godelstheory) [reply src/datasets/bigquery/clients_last_seen/reference.md:68]: Yep. added.

File: src/datasets/bigquery/clients_last_seen/reference.md (hunk 2 above)
Comment (godelstheory) [reply src/datasets/bigquery/clients_last_seen/reference.md:72]: Fixed.

**Pull request merged**